utils.py

```diff
@@ -0,0 +1,41 @@
+import shutil
+import os
+import yaml
+
+def init_nl(args):
+    # Define the path to the template inference.py file
+    inference_path = os.path.join(os.path.dirname(__file__), 'templates', 'inference.py')
+    config_path = os.path.join(os.path.dirname(__file__), 'templates', 'config.yaml')
+    
+    # Get the current working directory
+    root_dir = os.getcwd()
+    
+    # Define the path where you want to copy the template file
+    destination_path_inference = os.path.join(root_dir, 'inference.py')
+    config_path_inference = os.path.join(root_dir, 'config.yaml')
+
+    # Copy the template file to the current directory
+    shutil.copyfile(inference_path, destination_path_inference)
+    shutil.copyfile(config_path, config_path_inference)
+
+    print("initialized neura-launch project")
+
+def add_token(args):
```

Comment:
I feel it's better to add the token to the OS keyring than save it directly in a yaml file.

Reply:
totally can do that! this is also up to debate not sure what approach we want to go with... @rohansingh9001 shine ur light upon us